lib/spaceship/tunes/tunes_client.rb

```diff
@@ -632,13 +632,16 @@ def get_build_info_for_review(app_id: nil, train: nil, build_number: nil)
       r.body['data']
     end
 
-    def update_encryption_compliance(app_id: nil, train: nil, build_number: nil, encryption_info: nil, encryption: nil)
+    def update_encryption_compliance(app_id: nil, train: nil, build_number: nil, encryption_info: nil, encryption: nil, is_exempt: true, proprietary: false, third_party: false)
```

Comment:
Could you add a brief description of the newly added parameters in front of this method? You can do so using
```ruby
# @param is_exempt (Bool) Describes bla bla
```